src/components/CustomStatusBar/index.android.js

```diff
@@ -9,7 +9,6 @@ export default class CustomStatusBar extends React.Component {
     componentDidMount() {
         StatusBar.setBarStyle('dark-content', true);
         StatusBar.setBackgroundColor('transparent', true);
-        StatusBar.setTranslucent(true);
```

Comment:
If we remove `setTranslucent` here, should we also remove it from the comment above that mentions it?

Reply:
Oh yeah. Didn't notice it.